index.js

```diff
@@ -154,22 +136,22 @@ async function run() {
 
   if (trace) cmd.push(`--trace=${trace}`);
   if (outfile) cmd.push(`--outfile=${outfile}`);
-  core.info("Running in directory " + process.env.GITHUB_WORKSPACE);
+  core.info('Running in directory ' + fullWorkspacePath);
```

Comment:
Are these the only "real" changes in this file?

Reply:
this and lines `12-14`